bpf/mock/conntrack_vacant.h

```diff
@@ -0,0 +1,12 @@
+// Header of functions to be mocked in conntrack.h.
```

Comment:
For each of the hand-written files, please add the license. The common form looks like this for BPF code:

```
/* SPDX-License-Identifier: GPL-2.0 */
/* Copyright (C) 2021 Authors of Cilium */
```